writing_checks/README.md

```diff
@@ -468,6 +468,38 @@ whereas if it run on a system with ALPS it will be translated to
 ddt --offline aprun ...
 ```
 
+## Custom Launchers
+
+ReFrame permits the simple addition of custom scheduler launchers. A launcher that invokes `mpirun` can for example be added as follows:
+
+```python
+class MpirunLauncher(JobLauncher):
+    @property
+    def executable(self):
+        return 'mpirun'
+
+    @property
+    def fixed_options(self):
+        return [ '-np %s' % self.job.num_tasks ]
+```
+It will be translated to `mpirun -np ...`. While the definition of the property `executable` is obviously mandatory, the definition of `fixed_options` is optional; it defaults to no options. Note that the launcher class may use the information contained in `self.job`.
+
+A custom launcher as the above defined `MpirunLauncher` may be used in a regression test as follows:
+
+```python
+class MpirunTest(RegressionTest):
+    ...
+    def setup(self, system, environ, **job_opts):
+        super().setup(system, environ, **job_opts)
+        self.job.launcher = MpirunLauncher(self.job)
+```
+
+ReFrame will provide a collection of custom launchers (added uppon request). Currently it provides only one: a launcher for VisIt. The VisIt launcher can be used in a regression test the same way as the above `MpirunLauncher`; only the following import statement is required to make it available:
```

Comment:
I would convert the the VisIt to a link to the project's site. I would also add "Currently it provides only one: a launcher for the VisIt visualisation software".